make/dev.mk

```diff
@@ -46,3 +46,8 @@ else
 	release-please release-pr \
 	  --repo-url $(GITHUB_ORG)/$(GITHUB_REPOSITORY) --token "$$(gh auth token)"
 endif
+
+.PHONY: .envrc.local
+.envrc.local:
+	gojq --yaml-input --raw-output '.variables | to_entries | map("\(.key)=\(.value|tostring)")|.[]' < test/e2e/config/caren.yaml | envsubst > .envrc.local
```

Comment:
```suggestion
	gojq --yaml-input --raw-output '.variables | to_entries | map("export \(.key)=\(.value|tostring)")|.[]' < test/e2e/config/caren.yaml | envsubst >> .envrc.local
```

This allows us to have some content in the file already 

eg: 

```
03:34 PM  faiqus @ archlinux  ~/go/src/github.com/d2iq-labs/cluster-api-runtime-extensions-nutanix  shalin/local-envrc⚡? 
$ cat .envrc.local
export NUTANIX_USER="<REDACTED>"
export NUTANIX_PASSWORD="<REDACTED>"
export NUTANIX_STORAGE_CONTAINER_NAME="k8s"
export DOCKER_HUB_USERNAME="<REDACTED>"
export DOCKER_HUB_PASSWORD="<REDACTED>"
export NUTANIX_ENDPOINT="prismcentral.dev.ntnxsherlock.com"
export NUTANIX_PORT="9440"

export NUTANIX_PRISM_ELEMENT_CLUSTER_NAME="ganon"
export NUTANIX_INSECURE=false
export NUTANIX_SUBNET_NAME="sherlock_net"
export NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME="ubuntu-2204-kube-v1.28.7.qcow2"
export CONTROL_PLANE_ENDPOINT_IP="10.40.142.2"
export KUBERNETES_VERSION="v1.28.7"

03:34 PM  faiqus @ archlinux  ~/go/src/github.com/d2iq-labs/cluster-api-runtime-extensions-nutanix  shalin/local-envrc⚡? 
$ make .envrc.local 
direnv: loading ~/go/src/github.com/d2iq-labs/cluster-api-runtime-extensions-nutanix/.envrc
direnv: using devbox
./.envrc:49: warning: setlocale: LC_CTYPE: cannot change locale (en_US.UTF-8): No such file or directory
direnv: export +AMI_LOOKUP_BASEOS +AMI_LOOKUP_FORMAT +AMI_LOOKUP_ORG +AR +AS +CAPI_DIAGNOSTICS_ADDRESS +CAPI_INSECURE_DIAGNOSTICS +CC +CLUSTER_TOPOLOGY +CONFIG_SHELL +CONTROL_PLANE_ENDPOINT_IP +CXX +DETERMINISTIC_BUILD +DEVBOX_CONFIG_DIR +DEVBOX_INIT_PATH +DEVBOX_NIX_ENV_PATH_02dc8dc79ff6f450135cda448df9cd209d436b7f5272e2806ffd63d0eb5aa617 +DEVBOX_NO_ENVRC_UPDATE +DEVBOX_PACKAGES_DIR +DEVBOX_PATH_STACK +DEVBOX_PROJECT_ROOT +DEVBOX_SYSTEM_BASH +DEVBOX_SYSTEM_SED +DOCKER_HUB_PASSWORD +DOCKER_HUB_USERNAME +EXP_CLUSTER_RESOURCE_SET +EXP_KUBEADM_BOOTSTRAP_FORMAT_IGNITION +EXP_MACHINE_POOL +EXP_MACHINE_SET_PREFLIGHT_CHECKS +EXP_RUNTIME_SDK +HOST_PATH +IN_NIX_SHELL +KUBERNETES_VERSION +KUBERNETES_VERSION_DOCKER +KUBERNETES_VERSION_MANAGEMENT +LD +LD_LIBRARY_PATH +LIBRARY_PATH +NIX_BINTOOLS +NIX_BINTOOLS_WRAPPER_TARGET_HOST_x86_64_unknown_linux_gnu +NIX_BUILD_CORES +NIX_CC +NIX_CC_WRAPPER_TARGET_HOST_x86_64_unknown_linux_gnu +NIX_CFLAGS_COMPILE +NIX_ENFORCE_NO_NATIVE +NIX_HARDENING_ENABLE +NIX_LDFLAGS +NIX_STORE +NM +NODE_DRAIN_TIMEOUT +NUTANIX_ENDPOINT +NUTANIX_INSECURE +NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME +NUTANIX_PASSWORD +NUTANIX_PORT +NUTANIX_PRISM_ELEMENT_CLUSTER_NAME +NUTANIX_STORAGE_CONTAINER_NAME +NUTANIX_SUBNET_NAME +NUTANIX_USER +OBJCOPY +OBJDUMP +POD_CIDR +PYTHONHASHSEED +PYTHONNOUSERSITE +RANLIB +READELF +SERVICE_CIDR +SIZE +SOURCE_DATE_EPOCH +STRINGS +STRIP +_PYTHON_HOST_PLATFORM +_PYTHON_SYSCONFIGDATA_NAME +__DEVBOX_SHELLENV_HASH_02dc8dc79ff6f450135cda448df9cd209d436b7f5272e2806ffd63d0eb5aa617 +__ETC_PROFILE_NIX_SOURCED +__structuredAttrs +buildInputs +buildPhase +builder +cmakeFlags +configureFlags +depsBuildBuild +depsBuildBuildPropagated +depsBuildTarget +depsBuildTargetPropagated +depsHostHost +depsHostHostPropagated +depsTargetTarget +depsTargetTargetPropagated +doCheck +doInstallCheck +dontAddDisableDepTrack +mesonFlags +name +nativeBuildInputs +out +outputs +patches +phases +preferLocalBuild +propagatedBuildInputs +propagatedNativeBuildInputs +shell +shellHook +stdenv +strictDeps +system ~PATH ~PYTHONPATH ~XDG_DATA_DIRS

03:34 PM  faiqus @ archlinux  ~/go/src/github.com/d2iq-labs/cluster-api-runtime-extensions-nutanix  shalin/local-envrc⚡? 
$ cat .envrc.local
export NUTANIX_USER="<REDACTED>"
export NUTANIX_PASSWORD="<REDACTED>"
export NUTANIX_STORAGE_CONTAINER_NAME="k8s"
export DOCKER_HUB_USERNAME="<REDACTED>"
export DOCKER_HUB_PASSWORD="<REDACTED>"
export NUTANIX_ENDPOINT="prismcentral.dev.ntnxsherlock.com"
export NUTANIX_PORT="9440"

export NUTANIX_PRISM_ELEMENT_CLUSTER_NAME="ganon"
export NUTANIX_INSECURE=false
export NUTANIX_SUBNET_NAME="sherlock_net"
export NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME="ubuntu-2204-kube-v1.28.7.qcow2"
export CONTROL_PLANE_ENDPOINT_IP="10.40.142.2"
export KUBERNETES_VERSION="v1.28.7"
export AMI_LOOKUP_BASEOS=rocky-9.1
export AMI_LOOKUP_FORMAT=konvoy-ami-{{.BaseOS}}-release-?{{.K8sVersion}}-*
export AMI_LOOKUP_ORG=999867407951
export CAPI_DIAGNOSTICS_ADDRESS=:8080
export CAPI_INSECURE_DIAGNOSTICS=true
export CLUSTER_TOPOLOGY=true
export EXP_CLUSTER_RESOURCE_SET=true
export EXP_KUBEADM_BOOTSTRAP_FORMAT_IGNITION=true
export EXP_MACHINE_POOL=true
export EXP_MACHINE_SET_PREFLIGHT_CHECKS=true
export EXP_RUNTIME_SDK=true
export KUBERNETES_VERSION=v1.28.7
export KUBERNETES_VERSION_DOCKER=v1.29.4
export KUBERNETES_VERSION_MANAGEMENT=v1.29.4
export NODE_DRAIN_TIMEOUT=60s
export POD_CIDR=192.168.0.0/16
export SERVICE_CIDR=10.128.0.0/12
```

Reply:
multiple execution of `make .envrc.local` will keep adding to the file. Let me see if I can keep it unique+latest.

Reply:
You could always use multiple .envrcs and load them all.

Reply:
Thanks for easy suggestion. 2bc86cff7841fd33f7e5f5b24a71f9380209fbcc